### PR TITLE
chore(workflows): added GitHub Action for greeting

### PR DESCRIPTION
**Pull request by NarmeenNaveedAhmed**
## Description
This PR is to welcome new contributors to repository when they submit a pr or an issue.

## Motivation and Context
This is just to notify the contributor in the case of their first contribution.

## How Has This Been Tested?
-

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

## What is the Impact to Developers Using One App?
This is just to notify the contributor in the case of their first contribution.

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for submitting this issue.'
+        pr-message: 'Thank you for your contribution to this repository. We will be reviewing your PR. In the meantime please sign the CLA below.'
```
Comment (JamesSingleton) [inline .github/workflows/greetings.yml]: This message might come before or after the CLA message presents itself. We might want to reword this to `In the meantime, don't forget to sign the CLA.` or something of that nature.

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for submitting this issue.'
```
Comment (JamesSingleton) [inline .github/workflows/greetings.yml]: Can we add something about pull requests are welcome or something like this? `Thanks for opening your first issue. Pull requests are always welcome!` We want the OSS world to know that we are open to PRs.

Comment (NarmeenNaveedAhmed) [reply .github/workflows/greetings.yml]: I'll change that now! :)

File: .github/workflows/greetings.yml (hunk 1 above)
Comment (NarmeenNaveedAhmed) [reply .github/workflows/greetings.yml]: > ohhh got it, I'll change that now! :)
